pennylane/ops/qubit/parametric_ops.py

```diff
@@ -224,8 +224,20 @@ def compute_matrix(theta):  # pylint: disable=arguments-differ
         return qml.math.diag([p, qml.math.conj(p)])
 
     @classmethod
-    def _eigvals(cls, *params):
-        theta = params[0]
+    def compute_eigvals(cls, theta):  # pylint: disable=arguments-differ
+        """Canonical eigenvalues of the RZ operator.
```

Comment:
See comment above, if we want to be precise we'd probably have to refer to the Eigenvalues of the canonical matrix representation of this operator? Or does this not apply here?

Reply:
I'm not sure I understand :)

Reply:
I think I confused myself